includes/class-amp-story-post-type.php

```diff
@@ -564,6 +566,33 @@ public static function register_story_card_styling( WP_Styles $wp_styles ) {
 		);
 	}
 
+	/**
+	 * Export data used for Latest Stories block.
+	 */
+	public static function export_latest_stories_block_editor_data() {
+		$url = add_query_arg(
+			'ver',
+			wp_styles()->registered[ self::STORY_CARD_CSS_SLUG ]->ver,
+			wp_styles()->registered[ self::STORY_CARD_CSS_SLUG ]->src
+		);
+
+		/** This filter is documented in wp-includes/class.wp-styles.php */
+		$url = apply_filters( 'style_loader_src', $url, self::STORY_CARD_CSS_SLUG );
+
+		wp_add_inline_script(
+			AMP_Editor_Blocks::SCRIPT_HANDLE,
```

Comment:
So this is inside the `AMP_Story_Post_Type` class and adds the script to the one emitted by `AMP_Editor_Blocks` (`editor-blocks.js`), yet the actual usage of this data is in `amp-block-editor.js` which is enqueued by `AMP_Post_Meta_Box`.

Reply:
I guess I got the script handle wrong?

Reply:
If you change the script handle you don't need cc061291c5144e3afeeffa476e2d9576898fa02f and also not the `add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'export_latest_stories_block_editor_data' ), 100 );` part in `AMP_Story_Post_Type`.

Reply:
Is there a `editor-blocks.js` enqueued? I only see `amp-editor-blocks.js`

Reply:
Sorry, I meant `amp-editor-blocks.js`

Reply:
Oh, there s both an `amp-block-editor.js` _and_ an `amp-editor-blocks.js`.

Reply:
Done in 542e5850f95609c4fcff1aef422a27eeb88bc885